subsys/shell/shell_cmds.c

```diff
@@ -346,7 +346,7 @@ static int cmd_help(const struct shell *shell, size_t argc, char **argv)
 
 	shell_fprintf(shell, SHELL_NORMAL,
 		"Please press the <Tab> button to see all available commands.\n"
-		"You can also use the <Tab> button to prompt or auto-complete"
+		"You can also use the <Tab> button to prompt or auto-complete "
 		"all commands or its subcommands.\n"
```

Comment:
I think that our CI expects to move space to beginning of a new line not at the end of previous.